_includes/specials/taproot/en/20-activation.md

```diff
@@ -0,0 +1,66 @@
+Less than two weeks after the publication of this post, taproot is
+expected to activate at block {{site.trb}}.  We know what we expect to
+happen, and we also know about a few possible failure modes.
+
+The best and most likely outcome is that everything goes fine.  Nothing
+that happens should be visible to normal users.  Only those carefully
+monitoring their nodes or trying to create taproot transactions should
+notice anything.  At block 709,631 nearly all full nodes we're aware of
+will be enforcing the same consensus rules.  One block later, nodes
+running Bitcoin Core 0.21.1, 22.0, or related releases will be enforcing
+the additional [taproot][topic taproot] rules not enforced by earlier
+software releases.
+
+This carries a risk that earlier and later versions of node software
+will accept different blocks.  This happened during the activation of
+the [BIP66][] soft fork in 2015, resulting in a six-block chainsplit and
```

Comment:
Do we have a house style for "chainsplit" vs "chain split". We seem to have roughly the same count for both, but "chainsplit" seems a little inconsistent with our preferred "block chain".